Dockerfile

```
@@ -3,164 +3,107 @@ FROM elixir:1.14-otp-25 AS builder
+    apt-get clean && rm -rf /var/lib/apt/lists/* && \
+    sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+    locale-gen
 
 WORKDIR /app
 
-# Copy and extract the release
+# Copy and extract the release in one step
 COPY --from=builder /app/release.tar.gz /app/
-RUN mkdir -p /app/extracted && \
+COPY start.sh /app/start.sh
+RUN mkdir -p /app/extracted /app/data/cache /app/chart-output && \
```

Hadolint (2.12.0)

[warning] Use -print0/-0 or -exec + to allow for non-alphanumeric filenames.

(SC2038)